### ISSUE TITLE
Constraints are incorrectly marked as invalid in studio

### ISSUE THREAD

**Issue by pawelblaszczyk5** [commonalityco/commonality#337]
After updating commonality all the constraints are marked as invalid in the studio app. Every single connection is red 😄 

![image](https://github.com/commonalityco/commonality/assets/54995479/6302964a-4870-4e08-b3d6-20eea9c8e3c2)

However, when running CLI everything passes:

![image](https://github.com/commonalityco/commonality/assets/54995479/30c701e3-e480-458b-8ca1-937ece9ad333)

My constraints config is:

```json
{
	"constraints": {
		"*": { "disallow": ["application"] }
	}
}
```

Even if I don't mark any package with the "application" tag, it marks every constrain as invalid in graph. Removing the `constraints` completely works though. 

I've quickly setup a [minimal reproduction](https://github.com/pawelblaszczyk5/monorepo-commonality-minimal) for you. Screenshots are from my main repository, don't be surprised by different package names, but it's the same issue in minimal repo as well 😃

**Comment by alec-chernicki** [comment]
Thanks for opening this, I can repro on my end and I'll look into it this weekend!

**Comment by alec-chernicki** [comment]
This should now be fixed in commonality@0.2.60!
